### PR TITLE
Show Smaller Sub Map after Drawing Rectangle

### PR DESCRIPTION
**Pull request by wgranger**
This PR builds off #13 but opens the map detail modal after releasing the mouse and drawing a rectangle. After drawing a rectangle, the [modal should appear](https://projects.invisionapp.com/d/#/console/19544062/413573564/preview), with the area of interest contained within the map. 

The main goal here is to get the modal to open with the map cutout in view. I realize some items may be sized a bit odd on the modal, but I'm going to address resizing/repositioning some of those items in a separate PR. Also, a user should **not** be able to pan or zoom any of the Leaflet maps on the app, including the small one in this PR.

### CODE REVIEW

File: src/components/MapDetail/MapDetail.js [hunk 1]
```diff
@@ -77,17 +77,38 @@ export default function MapDetail() {
...
+          <Box
+            align='center'
+            background='gray'
+            border={{ color: 'kelp' }}
+            flex
+            justify='center'
+            style={{ position: 'relative' }}
+          >
+            <Map 
+              bounds={[coordinates.southWest, coordinates.northEast]}
```
Comment (mcbouslog) [inline src/components/MapDetail/MapDetail.js:102]: I'm getting an `Uncaught Error: Bounds are not valid.` error preventing the MapDetail/Default storybook from rendering? I think from this line, but not entirely sure.

**Review by mcbouslog — APPROVED** [review]
LGTM.

Rename `DrawingOverlap.spec.js` to Overlay with this or subsequent PR?

I edited the MapDetail story with MapPage to test. Up to you if want to add a MapPage or DrawingOverlay story in this PR or subsequently.

**Comment by wgranger** [comment]
Oops! That might've been a typo (`DrawingOverlap`). Either way, I'll change it now.

File: src/components/MapDetail/MapDetail.js (hunk 1 above)
Comment (wgranger) [reply src/components/MapDetail/MapDetail.js:102]: Good catch. I didn't have the appropriate default props for the component.

**Comment by wgranger** [comment]
@mcbouslog Changes made per your review. I also fixed a peculiar bug in the `DrawingOverlay`. When the "Draw Rectangle" button is inactive, a user is still able to click on the map and start an invisible rectangle. Then, when the user clicks to "Draw Rectangle," they'll notice the starting point has already been placed.

That shouldn't be the case. Now, a rectangle can only begin if the "Draw Rectangle" button is active.

**Review by mcbouslog — APPROVED** [review]
LGTM!

**Pull request merged**